server/pages/index.js

```diff
@@ -178,18 +191,28 @@ function getDefaultContext( request ) {
 		bodyClasses.push( 'pride' );
 	}
 
-	if ( config( 'rtl' ) ) {
+	if ( isRTL ) {
 		bodyClasses.push( 'rtl' );
 	}
 
+	if ( request.context.sectionCss ) {
+		if ( isRTL ) {
+			sectionCss = getHashedUrl( 'sections/' + request.context.sectionCss + '-rtl.css' );
```

Comment:
So, if I understand this correctly, we're going to read the css file and compute the hash for each request. That doesn't sound optimal.

Reply:
The hash should be added to the filename imo